### ISSUE TITLE
Add aria-label's to buttons/links without text

### ISSUE THREAD

**Issue by felixwetell** [Linda-Carlstad/lindacarlstad.se#109]
Buttons and links without a text should contain a aria-label to let screen readers know what that element does.

**Comment by felixwetell** [comment]
Sponsor images on first page should have aria-labels.
